arrow-docs/docs/arrow/typeclasses/functorfilter/README.md

```diff
@@ -9,6 +9,50 @@ permalink: /arrow/typeclasses/functorfilter/
 
 
 
+`FunctorFilter` is a typeclass that abstracts over sequential execution of code.
```

Comment:
The tagline is copied from Monad I believe :D

Reply:
Yes! :D Everything is copy/paste.

Reply:
I mean, it needs to be updated for this typeclass. It's what's left to go to production 🙏